relay-aws-extension/src/aws_extension.rs

```diff
@@ -0,0 +1,193 @@
+use actix::actors::signal::{Signal, SignalType};
+use actix::prelude::*;
+use failure::Fail;
+use relay_system::Controller;
+use reqwest::blocking::Client;
+use reqwest::StatusCode;
+use serde::Deserialize;
+use std::collections::HashMap;
```

Comment:
Please group imports like so:

```suggestion
use std::collections::HashMap;

use actix::actors::signal::{Signal, SignalType};
use actix::prelude::*;
use failure::Fail;
use reqwest::{blocking::Client, StatusCode};
use serde::Deserialize;

use relay_system::Controller;
```